options.go

```diff
@@ -309,6 +312,39 @@ func LabelLimitValueSize(size int) Option {
 	}
 }
 
+// SelfEvictDisablePing configures if ringpop should disable actively pinging
+// other members when it evicts itself from the network. It is not advised to
+// disable the ping as it might cause the suspicion time to kick in as chances
+// are high that the node will be shutdown before its self-eviction is gossiped.
+// Only use this setting if you explicitly require no extra network traffic
+// during shutdown.
+func SelfEvictDisablePing(disabled bool) Option {
```

Comment:
OK, it's just that I can't imagine a scenario where this would ever be a requirement: "Only use this setting if you explicitly require no extra network traffic during shutdown".

If there is no imaginable scenario where someone would use this option, then I'd remove it, as it means one less thing we have to support.